spacemacs/packages.el

```diff
@@ -3708,10 +3708,15 @@ one of `l' or `r'."
               (,(listify-key-sequence
                  (kbd dotspacemacs-emacs-leader-key)) . "Spacemacs root")))
       (nconc which-key-prefix-title-alist spacemacs/prefix-titles)
+      (pcase dotspacemacs-which-key-position
+        ('right (which-key-setup-side-window-right))
+        ('bottom (which-key-setup-side-window-bottom))
+        ('right-then-bottom (which-key-setup-side-window-right-bottom)))
```

Comment:
With single quotes I get `Error (use-package): which-key :init: Unknown upattern `(quote right)'`. When I change the single quote to backticks, it works. What am I missing?